mlflow/tracking/client.py

```diff
@@ -208,7 +208,7 @@ def set_terminated(self, run_id, status=None, end_time=None):
                        Defaults to "FINISHED".
         :param end_time: If not provided, defaults to the current time."""
         end_time = end_time if end_time else int(time.time() * 1000)
-        status = status if status else "FINISHED"
+        status = status if status else RunStatus.to_string(RunStatus.FINISHED)
```

Comment:
flipping existing usage of free flowing strings to pre-defined constants. ^^this^^ is one example. More further down..